blogs/data-mover/data-mover-1.3/data-mover-1.3.0-overview.md

```diff
@@ -0,0 +1,230 @@
+
+Now working from the Data Mover plugin the CSI snapshot is mounted from the Node-Agent. The DataUpload Controller then works with Kopia ( the uploader ) to move the object off cluster to the Unified Repo Backup repository off cluster.  The status is once again reconciled and the backup CR is moved to complete.<br>
+
+Users can see the DataUpload objects move to a terminal status of either `Completed`, `Failed` or `Canceled`
+Once the object has been uploaded any intermediate objects like the VolumeSnapshot and VolumeSnapshotContents will be removed. Finally the backup object status will be updated with it's terminal status.
+
+</div>
+<hr>
+
+<p dir="auto"><img alt="backup-13-workflow" src="backup-13-workflow.png" width="850" /></p>
+
+<div>
+<hr>
+A more in depth visualization of the backup workflow with Data Mover is found below.
+<hr>
+</div>
+
+<p dir="auto"><img alt="data-mover-13-backup-sequence" src="data-mover-13-backup-sequence.png" width="850" /></p>
+
+<h2>Restore Process</h2>
+<div>
+A user creates a restore CR, no additional data mover options or parameters are required. Velero calls the RIA V2 api to create a PV and PVC in the protected namespace (openshift-adp). The backup status will move from `New` to `InProgress`.<br><br>
+
+The data from backup is queried from the remotely stored DataUpload CR and written to the local ConfigMaps.  A ConfigMap is created for each PV to be restored. These Configmaps are temporary objects that are deleted upon the restore's workflow completion. The ConfigMap stores vital information like the Repo Snapshot ID or VolumeSnapshotContent name.  The data stored in the ConfigMap is then used to build the DataDownload CR spec. 
+
+The DataMover plugin creates the DataDownload CR and DataDownload Controller reconciles on the CR.  The Node-Agent begins the download the backed up PV data from S3. 
+
+As the data from the backup is downloaded via DataDownload Controller via Kopia the target volume is marked as not ready. In order  to prevent the volume from binding the spec.VolumeName set to empty (""). The status of the download can be viewed from the DataDownload CR object as `Accepted`, `Prepared`, or `InProgress`.  Similarly with the Data Mover backup process a user may find temporary objects (i.e., pods, PVCs, PVs) created in the protected namespace (openshift-adp) during this step.
+
+Once the DataDownload is in a terminal status `Completed`, the target PVC should have been created in the target user namespace and waiting for binding.  The PV's claim reference is written to the target PVC in the target user namespace and the PVC will be immediately bound to the target PV.  
+
+Please note the advantage of reusing the same cluster scoped PV throughout the restore eliminates the need to recopy the PV data from the protected namespace to the application namespace as was the requirement for OADP-1.2
```

Comment:
This is incorrect, we did not re-copy data from protected ns to application ns in OADP 1.2 datamover, we used to get the snaphandle from volsync VolumeSnapshotContent and use it in our modified CSI plugin's VS and PVC creation in application NS. Overall on a high-level the OADP 1.2 datamover and VBDM work in the same way, its just that VBDM is more tightly coupled with Velero controller and OADP 1.2 datamover was loosly coupled and dependent on VolSync controller.